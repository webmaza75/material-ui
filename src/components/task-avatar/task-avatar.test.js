import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import TaskAvatar from './task-avatar.jsx';

describe(`TaskAvatar`, () => {
  it(`TaskAvatar renders correctly`, () => {
    const renderer = new ShallowRenderer();
    const result = renderer
      .render(<TaskAvatar />);

    expect(result).toMatchSnapshot();
  });
});