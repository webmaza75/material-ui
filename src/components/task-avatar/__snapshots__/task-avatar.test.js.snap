// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskAvatar TaskAvatar renders correctly 1`] = `<ForwardRef(WithStyles) />`;
